Cookbook/Technical-Documents/JIRA.md

```diff
@@ -0,0 +1,38 @@
+# JIRA 
+### Overview
+JIRA is the project management tool we use in all development squads at Babylon. Sometimes we make use of other tools like Trello for certain purposes but always one common thing between squads is JIRA. 
+JIRA is a really powerful and customisable tool - how we use depends on the squad. Some squads use a simple setup with few statuses, others use advanced workflows with custom fields and many statuses. Although we allow customisation, there are a couple of things that remain shared across all projects which will be covered by the next sections. **One very important thing common for all the squads is that JIRA is our main source of truth and we should keep it as clean and up-to-date as possible**.
+
+### How we’re using it
+We don’t have one common methodology (Scrum, Kanban etc) so selecting one depends on the squad. Each squad sets its own rules of working so make sure to familiarise yourself with those when you join your squad.
+
+Even that each project can be different we share some of the statuses between each of them to allow our automation to do the hard work of moving tickets between columns (setting up this automation requires some work - check `Automation` section for details). Keep in mind that even we have automation in place, sometimes we have to do it manually.
+
+Common statuses:
+* To Do / Ready For Development 
+* In Progress
+* Peer Review
+* Awaiting Build
+* QA
+
+### SSDLC and JIRA
+Babylons follows [Secure Software Development Lifecycle (SSDLC)](https://engineering.ops.babylontech.co.uk/docs/standards-ssdlc/) document that describes how software is built and run. One of the important rules is to have a JIRA ticket for all code changes we make. To ensure that, we do have `babylon/ssdlc-jira`  check for each PR we open. The only thing we have to do to comply with this requirement is to insert JIRA ticket in the Pull Request title on GitHub. Thanks to this, our tooling we’ll be able to pick up all merged pull requests between releases and create [CRP ticket](https://engineering.ops.babylontech.co.uk/docs/cicd-deployments/) with all JIRA tickets linked there when necessary.
```

Comment:
```suggestion
Babylons follows [Secure Software Development Lifecycle (SSDLC)](https://engineering.ops.babylontech.co.uk/docs/standards-ssdlc/) document that describes how software is built and run. One of the important rules is to have a JIRA ticket for all code changes we make to ensure that we do have `babylon/ssdlc-jira` check for each PR we open. The only thing we have to do to comply with this requirement is to insert JIRA ticket in the Pull Request title on GitHub. Thanks to this, our tooling will be able to pick up all merged pull requests between releases and create a [CRP ticket](https://engineering.ops.babylontech.co.uk/docs/cicd-deployments/) with all JIRA tickets linked there when necessary.
```

Reply:
I see you've modified this sentence to `One of the important rules is to have a JIRA ticket for all code changes we make to ensure that we do have `babylon/ssdlc-jira` check for each PR we open.` but I don't think it makes sense :)

Reply:
GitHub messed up my suggestion 😬 let me check again